.github/workflows/portage-stable-packages-list

```diff
@@ -500,6 +500,7 @@ sys-apps/texinfo
 sys-apps/usbutils
 sys-apps/util-linux
 sys-apps/which
+sys-apps/zram-generator
```

Comment:
This probably does not belong to this PR, right? I don't see the package being added here.

Reply:
removed